types/koa/index.d.ts

```diff
@@ -456,7 +457,25 @@ declare class Application<
     silent: boolean;
     keys: Keygrip | string[];
 
-    constructor();
+    /**
+     *
+     * @param {object} [options] Application optionsJake, 2 years ago: • feat: accept options in the Application const…
```

Comment:
@vnoder I think there is accidentally added text here, if I compare it with https://github.com/koajs/koa/blob/698ce0afbfac6480400625729a4b8fc4b4203fdc/lib/application.js#L39